src/app/tests/suites/certification/ci-pics-values

```diff
@@ -750,6 +750,27 @@ PIXIT.LWM.MODE_CHANGE_OK=1
 
 LWM.S.M.CAN_TEST_MODE_FAILURE=1
 
+
+# OVEN MODE CLUSTER
+OM.S=1
+OM.S.A0000=1
+OM.S.A0001=1
+OM.S.A0003=1
+OM.S.A0002=1
+
+#Feature
+OM.S.F00=1
+
+#Command
+OM.S.C00.Rsp=1
+OM.S.C01.Tx=1
+
+#PIXIT
```

Comment:
These can be removed - pics only in this file. Realistically, this file is terrible anyway, but it'll give us less to clean up.